### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by markashleybell** [JoshClose/CsvHelper#2087] [General]
Hi all, 

I just wanted to drop in here and say a huge thank you to @JoshClose—and everyone else who has contributed to this library—for all your hard work.

I use CSVHelper a _lot_ in my own projects, and every time I do it's easy, fast and a joy to build things with; reading or writing CSV data is something I just no longer need to worry about. And as someone with a lot of experience in attempting to manually parse CSV data back in the day, that is an absolute gift 😉

So, thanks again! And hopefully I can contribute at some point in the future, to help continue all the good work on this great bit of software.

**Comment by JoshClose** [comment]
Awesome, you're welcome, and thanks for the contribution! I'm glad you've found this useful and more importantly easy to use. I started out by creating an API I liked and figured out the implementation details later. I've tried to keep that in mind over the years. 

CSV is easy to parse until it's not. Then you find almost every system out there does something slightly different and doesn't conform to RFC 4180.

Thanks!

**Reply by markashleybell** [reply]
> CSV is easy to parse until it's not

Yep! Luckily, thanks to this library I can now mostly just leave all of that to the experts. 😁
